packages/office-ui-fabric-react/src/components/Coachmark/Coachmark.tsx

```diff
@@ -98,33 +134,69 @@ export class Coachmark extends BaseComponent<ICoachmarkTypes, ICoachmarkState> {
     };
   }
 
+  private get _beakDirection(): BeakDirection {
```

Comment:
I'd recommend using rectangleEdge from positioning. Then implement a public "getOppositeEdge" function in positioning which would look like 
```
export function getOppositeEdge(edge: RectangleEdge): RectangleEdge {
    return edge * -1;
} 
```